utils/src/mesh.rs

```diff
@@ -13,6 +14,8 @@ pub trait KaboodleMesh {
 
     /// Create a new entry for a Kaboodle peer network and add ourselves to the mesh.
     async fn start(&mut self) -> Result<(), KaboodleError>;
+    /// Remove this peer from the mesh.
```

Comment:
Total nit but maybe consider `Remove ourselves from the mesh` so it's more obvious that there is no way to remove forcibly an other peer from the mesh.